util/opentelemetry-util-http/setup.cfg

```diff
@@ -40,3 +40,7 @@ packages=find_namespace:
 
 [options.packages.find]
 where = src
+
+[options.entry_points]
+opentelemetry_instrumentor =
```

Comment:
Compared to the original PR version with a separate package for this:
On the minus side, this now means that everyone depending on util-http will get the httplib.client instrumentation, even if they don't use it. The overhead should be small though. On the plus side, there is no new package.

Reply:
The benefit of not having a new package outweighs the small overhead I feel :)